dgv/monitoring/DAG-notification.py

```diff
@@ -511,6 +548,11 @@ def publish_mattermost(ti):
         python_callable=alert_if_awaiting_spam_comments,
     )
 
+    alert_if_awaiting_spam_comments = PythonOperator(
```

Comment:
What was that not here?

Reply:
Forgot to modify after copy/paste 🙃 thanks for the alert